timelock/program/src/state/governance_voting_record.rs

```diff
@@ -0,0 +1,106 @@
+use super::UNINITIALIZED_VERSION;
+use arrayref::{array_mut_ref, array_ref, array_refs, mut_array_refs};
+use solana_program::{
+    program_error::ProgramError,
+    program_pack::{IsInitialized, Pack, Sealed},
+    pubkey::Pubkey,
+};
+
+/// STRUCT VERSION
+pub const GOVERNANCE_VOTING_RECORD_VERSION: u8 = 1;
+/// Governance Voting Record
+#[derive(Clone, Debug, Default, PartialEq)]
+pub struct GovernanceVotingRecord {
+    /// proposal
+    pub proposal: Pubkey,
```

Comment:
`proposal` is actually a `timelock_set`

Reply:
TimelockSet was renamed to Propsoal
TimelockConfig was renamed to Governance